Makefile

```diff
@@ -79,7 +79,7 @@ build: ## Builds all containers or the one specified (args: n={service name}, p=
 		-f docker-compose.override.yml \
 		-f ./db/kafka/docker-compose.yml \
 		-f ./services/docker-compose.yml \
-		$(if $(p),--profile $(p),--profile $(n)) \
+		$(if $(p),--profile $(p),$(if $(n),--profile all,)) \
```

Comment:
Fixing a number of docker-compose errors that make it hard to run containers with our make commands.